.github/workflows/test.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "18"
+          node-version: "21"
```

Comment:
Ideally, we should address these types of changes in their own separate PR.

Reply:
I agree, but the test were failing because of the node version. So either we update node in this PR or merge with failing tests.